apps/backend/.env.example

```diff
@@ -25,6 +25,14 @@ JWT_SECRET=<SECRET_PHRASE>
 JWT_EXPIRATION_TIME=24hr
 JWT_ALGORITHM=HS256
 
+#
+# AWS
+#
+BUCKET_NAME=<BUCKET_NAME>
```

Comment:
Don't forget to notify everyone in chat about this when these are pushed to main

Reply:
Yeah, I'll notify all and send a useful tutorial about what you should do, to get these variables